<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDlg</class>
 <widget class="QDialog" name="SettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="shapesLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Select playing shapes (more than 3):</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shapeCheckBox_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>70</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="shape_0.png"/>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shapeCheckBox_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>70</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_1.png</normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shapeCheckBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>70</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_2.png</normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shapeCheckBox_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>54</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_3.png</normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shapeCheckBox_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>55</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_4.png</normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shapeCheckBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>70</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_5.png</normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shapeCheckBox_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_6.png</normaloff>../build-Tetris-Desktop_Qt_5_8_0_MinGW_32bit-Debug/debug/shape_6.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="SpeedLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>238</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Speed level:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>239</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingOkPB">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingsCancelPB">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="Line" name="line0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="changeColorChkBox">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>300</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Change color </string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>342</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
